src/transformers/modeling_flax_pytorch_utils.py

```diff
@@ -254,7 +254,10 @@ def convert_pytorch_sharded_state_dict_to_flax(shard_filenames, flax_model):
     for shard_file in shard_filenames:
         # load using msgpack utils
         pt_state_dict = torch.load(shard_file, weights_only=is_torch_greater_or_equal_than_1_13)
-        pt_state_dict = {k: v.numpy() for k, v in pt_state_dict.items()}
+        weight_dtypes = {k: v.dtype for k, v in pt_state_dict.items()}
```

Comment:
Could you please rebase onto `main` as there have been some changes to this file!

Reply:
changes merged